thirdparty/configure-native-deps.sh

```diff
@@ -6,7 +6,7 @@
 # Copy-paste the entire script into http://shellcheck.net to check.
 ####
 
-locations="/lib /lib64 /usr/lib /usr/lib64 /usr/lib/i386-linux-gnu /usr/lib/x86_64-linux-gnu /usr/lib/arm-linux-gnueabihf/ /usr/lib/mipsel-linux-gnu /usr/local/lib /opt/lib /opt/local/lib /app/lib"
+locations="/lib /lib64 /usr/lib /usr/lib64 /usr/lib/i386-linux-gnu /usr/lib/x86_64-linux-gnu /usr/lib/arm-linux-gnueabihf /usr/lib/aarch64-linux-gnu /usr/lib/powerpc64le-linux-gnu /usr/lib/mipsel-linux-gnu /usr/local/lib /opt/lib /opt/local/lib /app/lib"
```

Comment:
This is becoming a bit silly now. How about this to cut down on the variants:

```sh
target_triplet="$(make -v | sed -n '2s/.* //p')"
locations="/lib /lib64 /usr/lib /usr/lib64 /usr/lib/${target_triplet} /usr/local/lib /opt/lib /opt/local/lib /app/lib"
```

Reply:
there's no guarantee that they will match-up. For example on my Ubuntu machine the triplet is `x86_64-linux-gnu` but the above will extract `x86_64-pc-linux-gnu` from `make -v` which isn't correct.

Reply:
we could try `gcc -v` which appears to show correctly in the `Target` field for my system. Could someone with other architectures, notably an arm and non-Ubuntus, check whether that fits their boxes, too?

Reply:
The current setup will take the first thing it finds without prioritizing based on the current arch, so what if we just loop over `$(ls /usr/lib*/*/liblua*.so*)` or something slightly smarter instead?

Reply:
Relying on gcc isn't such a good idea, we don't require gcc for anything else and so can't be sure it's even installed. But pchote's wildcard approach is ok with me.

Reply:
Ideally, shouldn't we be using `pkg-config` to solve this?

Reply:
At least on Debian/Ubuntu, the .pc file is part of the -dev package, not the regular lib package.

Reply:
IMO we'd be best merging this so @diddledan can get on with packaging, and then we can find a solution in our own time.

Reply:
yes, let's.